apis/google/gmail/code/sendEmail.py

```diff
@@ -20,6 +21,14 @@ def main():
     if message is None:
         raise ValueError("Email message must be set")
 
+#     if forwarded:
+    #         subject = f'Fwd: {subject}'
+    #         message = f'''---------- Forwarded message ---------
+    # From: {to_emails}
+    # {message}'''
+    #         test = f'''
+    #         To: {to_emails}'''
+
```

Comment:
Can this be removed? If so, looks like the `forwarded` variable can be removed, too.

Reply:
oh, yes forgot to remove that when i was moving the changes into a context tool.